README.markdown

```diff
@@ -90,6 +90,12 @@ Synopsis
         return ngx.exit(444)
     end
 
+    local ngx_exit = ngx.worker.exiting()
+    if ngx_exit then
+        ngx.log(ngx.ERR, "closing open websockets - nginx worker exiting")
+        wb:send_close()
+    end
```

Comment:
Also, checking for the worker exists is not needed at all in this code example since there is no Lua loop here and the check is useless.